services/hacker.service.js

```diff
@@ -213,6 +214,17 @@ function getStats(hackers) {
             : 1;
         const age = hacker.accountId.getAge();
         stats.age[age] = stats.age[age] ? stats.age[age] + 1 : 1;
+
+        const applicationDate = hacker._id
+            .getTimestamp()
+            .toISOString()
+            .slice(0, 10);
```

Comment:
Could you add some commenting to document what the .slice(0,10) does?

Reply:
done